app/screens/image_preview/image_preview.js

```diff
@@ -359,7 +367,7 @@ export default class ImagePreview extends PureComponent {
         return null;
     };
 
-    renderOtherItems = (index) => {
+    renderOtherItems = memoizeResult((index) => {
```

Comment:
Why are we memoizing this? This doesn't seem needed here

Reply:
Does this actually make a difference? Rendering this 3 times seems minimal in terms of performance

Reply:
@hmhealey it does, cause rendering 3 times will then perform 3 requests to get the file and set it in the device file cache

Reply:
@hmhealey do you mind taking a try at this? I'm currently held up in other matters and I don't think I'll have the time to address this feedback.

let me know

Reply:
I can try to find some time next week, but I might not have time to get to this for 1.17

Reply:
One more thing, I think this is not covering the case for inline images/image previews and opengraph.

I'm going to try and take another stab at this between today and tomorrow